tests/test_requests.py

```diff
@@ -285,6 +285,23 @@ def test_request_cookies():
     assert response.text == "Hello, cookies!"
 
 
+def test_invalid_cookie():
+    async def app(scope, receive, send):
+        request = Request(scope, receive)
+        if not request.cookies:
+            response = Response("ok", media_type="text/plain")
+        else:
+            response = Response("not", media_type="text/plain")
+        await response(scope, receive, send)
+
+    client = TestClient(app)
+    response = client.get("/", cookies={"invalid/cookie": "test", "valid": "test2"})
+    assert response.text == "ok"
+
+    response = client.get("/", cookies={"valid": "test2"})
+    assert response.text == "not"
```

Comment:
Nits:

- The fact that "ok" corresponds to "cookies are invalid" is a bit counter-intuitive to me.
- We already test the valid case, so we can probably focus this test on "what happens if we pass invalid cookies?".

For example…

```suggestion
def test_invalid_cookie():
    async def app(scope, receive, send):
        request = Request(scope, receive)
        response = JSONResponse({"cookies": request.cookies})
        await response(scope, receive, send)

    client = TestClient(app)
    response = client.get("/", cookies={"invalid/cookie": "test", "valid": "test2"})
    assert response.json() == {"cookies": {}}
```